src/code_gen.rs

```diff
@@ -42,7 +42,7 @@ pub fn build_entrypoint(
         entrypoint_content += "const container = document.getElementById('root');\n";
         entrypoint_content += "hydrateRoot(container, <Entrypoint />);\n";
     } else {
-        entrypoint_content += "import { renderToString } from 'react-dom/server';\n";
+        entrypoint_content += "import { renderToString } from 'react-dom/server.edge';\n";
```

Comment:
AFAIK the implementation of renderToString is the same in edge & non-edge server libraries. Edge is more applicable for `renderToReadableStream`. What's the rational for doing the switch here?

Reply:
@piercefreeman this is the reason I switched to the edge version https://github.com/facebook/react/issues/31827